fvtest/gctest/GCConfigTest.cpp

```diff
@@ -44,10 +44,15 @@ const char *gcTests[] = {"fvtest/gctest/configuration/sample_GC_config.xml",
                                 "fvtest/gctest/configuration/test_system_gc.xml",
                                 "fvtest/gctest/configuration/gencon_GC_config.xml",
                                 "fvtest/gctest/configuration/gencon_GC_backout_config.xml",
```

Comment:
Is the backout test a gencon test as well?

Reply:
Yep. I've redone these ifdefs.